individual/individual.js

```diff
@@ -1,25 +1,154 @@
 "use strict";
 
 //TODO: CHANGE limit on returned properties in function propertyTypeQuery()
-const DATASOURCE = 'j2j3-acqj' // ? '75rg-imyz'
```

Comment:
i think the data source needs to be switched for that soda-js to work properly, not sure why this slipped by.  'j2j3' should be switched for '75rg'.  i think soda-js wants the soda api v2 and 'j2j3' is served through soda api v1.